### DISCUSSION TITLE
The issue of material loss after lightweight rvt files

### DISCUSSION THREAD

**Discussion by wwr1982** [iTwin/itwinjs-core#6660] [Q&A]
I used iTwin snapshot software to lighten an rvt file with materials and found that the lightweight material is missing. What should I do to ensure that the materials in the rvt file are not lost? thanks!
The following is the effect of browsing in Revit before conversion:

![111111](https://github.com/iTwin/itwinjs-core/assets/20233157/28f4bd59-ab48-4517-a7ba-b13ced1d9040)

The following is the effect of lightweight browsing in itwins:
![222222](https://github.com/iTwin/itwinjs-core/assets/20233157/d02b478f-dfd6-4874-91f9-74b8ed7fbad6)

The following is the rvt file:
[aRVT.zip](https://github.com/iTwin/itwinjs-core/files/15073399/aRVT.zip)

**Comment by pmconne** [comment]
@timlawrence-bentley @jffmarker @abeesh

**Comment by jffmarker** [comment]
@pmconne , this seems like a render mode limitation... the screenshot looks like "solid fill", right? That doesn't seem to use or support textures.

![image](https://github.com/iTwin/itwinjs-core/assets/8006049/c62b368b-bd73-404e-b149-26e6916765f7)
vs.
![image](https://github.com/iTwin/itwinjs-core/assets/8006049/a544afdd-a8a2-4805-9c4c-a6cbeddab0bb)

@wwr1982, are you able to see the texture when using a different render mode such as "modeling"?

**Reply by wwr1982** [reply]
Thank you, I switch to smooth mode can display the material.
